website/docs/en-US/progress.md

```diff
@@ -182,6 +182,29 @@ You also can specify `type` attribute to `dashboard` to use dashboard progress b
 ```
 :::
 
+### Indeterminate progress
+
+:::demo Usa `indeterminate` attribute to set indeterminate progress. And use `speed` attribute to change the animation duration.
```

Comment:
Typo, `Use`
Maybe change this line to
**Use `indeterminate` attribute to set indeterminate progress, with `speed` to control the animation speed**